test/test-udp-send-ex.c

```diff
@@ -0,0 +1,258 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
```suggestion
/* Copyright libuv project contributors. All rights reserved.
```
(new files should use the new copyright boilerplate)